app/src/debug/java/io/reark/rxgithubapp/utils/StethoInstrumentation.java

```diff
@@ -73,18 +70,8 @@ void initStetho() {
     @NonNull
     public OkHttpClient decorateNetwork(@NonNull final OkHttpClient httpClient) {
         Preconditions.checkNotNull(httpClient, "Http Client cannot be null.");
-
-        addInterceptor(httpClient, interceptor);
-
-        return httpClient;
+        return httpClient.newBuilder().addNetworkInterceptor(new StethoInterceptor()).build();
```

Comment:
Is there a reason for not injecting the interceptor but creating it inside? It seems more coupled and less testable